openlibrary/templates/books/custom_carousel.html

```diff
@@ -1,5 +1,8 @@
 $def with(books=[], title="", url="", key="", min_books=1, load_more=None, test=False)
 
+$def sponsor_me_carousel():
```

Comment:
In order for this method to be accessible to the template, we'd need to use the `@public` decorator :)